### DISCUSSION TITLE
New Tab = Level 0

### DISCUSSION THREAD

**Discussion by TipTap12** [Able-Bear-Software/tab-nodes-tree#29] [Ideas / Feature Request]
I would like to request a feature addition to the application regarding the behavior of newly opened tabs. Specifically, I would appreciate the ability for new tabs opened manually from the toolbar to always appear as tabs at the top (level zero), rather than as tabs of the same level as the previous one.

This functionality would greatly enhance usability and improve workflow when working with multiple tabs, ensuring a more organized and efficient experience.

Thank you for considering this feature request, and I look forward to any updates on its implementation.

Best regards

![image](https://github.com/user-attachments/assets/bf57e271-8709-470f-9c8c-3dff046554bd)

**Comment by zeroarst** [comment]
@TipTap12  Thank you for your feedback! This should be straightforward to implement. I’ll add a new setting in options page.

**Reply by zeroarst** [reply]
#30

**Comment by zeroarst** [comment]
By the way, in the mean time, you could right click on a tab node and select "Create new tab at same level".

**Comment by zeroarst** [comment]
The setting should be there in 1.4.0 now. #30 
Feel free to open a new thread if any issues.
